html/browsers/browsing-the-web/navigating-across-documents/multiple-globals/context-for-websocket.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="resources/context-helper.js"></script>
+<script>
+window.scriptToRun = `
+const result = {};
+try {
+  const ws = new WebSocket('foo');
```

Comment:
This ends up creating the `WebSocket` object in the incumbent realm I think, which then ends up being its relevant realm.

Reply:
In general the way of testing these subtleties for constructor APIs like WebSocket or Worker are quite different than how you'd test them for methods like location.assign(). So I think following the pattern from https://github.com/web-platform-tests/wpt/tree/master/workers/multi-globals would work best.